Patch/BeatlessPatch.cs

```diff
@@ -0,0 +1,23 @@
+using BaboonAPI.Hooks.Tracks;
+using HarmonyLib;
+using TrombLoader.CustomTracks;
+
+namespace TrombLoader.Patch;
+
+[HarmonyPatch]
+public class BeatlessPatch
+{
+    [HarmonyPatch(typeof(GameController), nameof(GameController.Start))]
+    [HarmonyPostfix]
+    static void PatchBeatless(GameController __instance)
+    {
+        var track = TrackLookup.lookup(GlobalVariables.chosen_track);
+        if (track is CustomTrack ct)
+        {
+            if (ct.beatless)
+            {
+                __instance.goBeatless();
+            }
+        }
+    }
+}
```

Comment:
Rather than doing a harmony patch for this we can actually use the LoadedCustomTrack callbacks.
Either [`LoadBackground`](https://github.com/tc-mods/TrombLoader/blob/main/CustomTracks/CustomTrack.cs#L128) or [`SetUpBackgroundDelayed`](https://github.com/tc-mods/TrombLoader/blob/main/CustomTracks/CustomTrack.cs#L133) are useable here.